api/kyverno/v1/utils.go

```diff
@@ -27,6 +29,10 @@ func ToJSON(in apiextensions.JSON) *apiextv1.JSON {
 	return &out
 }
 
+func IsJSONEmpty(in *apiextv1.JSON) bool {
```

Comment:
Are we optimising the way we detect empty json here ? 🤔

Reply:
Yeah I found one of the main uses is to check emptiness. We used to unmarshal every time to check that.

Reply:
Again this is not the correct approach...

Reply:
It's definitely not to introduce such micro optims.

> We used to unmarshal every time to check that

Marshal less then.

Reply:
We are not marshaling here. Rather it's stored in its raw format and every time we need to access its value we unmarshal. Ideally we would unmarshal only once and store it. If this is what you mean then I get it I'm trying to get this to work.